compressor/parser/beautifulsoup.py

```diff
@@ -1,42 +1,46 @@
 from __future__ import absolute_import
 from django.core.exceptions import ImproperlyConfigured
-from django.utils import six
 from django.utils.encoding import smart_text
 
-from compressor.exceptions import ParserError
 from compressor.parser import ParserBase
-from compressor.utils.decorators import cached_property
 
 
 class BeautifulSoupParser(ParserBase):
 
-    @cached_property
-    def soup(self):
+    def __init__(self, content):
+        super(BeautifulSoupParser, self).__init__(content)
         try:
-            if six.PY3:
-                from bs4 import BeautifulSoup
-            else:
+            from bs4 import BeautifulSoup
+            self.use_bs4 = True
+            self.soup = BeautifulSoup(self.content, "lxml")
```

Comment:
i put lxml in there because of a warning of beautifulsoup, see here: http://www.crummy.com/2015/06/29/0, but there is no specific reason why i chose lxml, i think i did it because the warning said so. i could put html.parser in there, if you want. not specifying the parser will cause beautifulsoup to choose lxml, html5lib and then html.parser, in that order, and output the warning.

regarding the import error: http://www.crummy.com/software/BeautifulSoup/bs4/doc/#specifying-the-parser-to-use: "If you don’t have an appropriate parser installed, Beautiful Soup will ignore your request and pick a different parser."
so, no import error. i also briefly tested this by removing lxml from one of the tox envs.